_integration-tests/tests/autoupdate-msg_test.go

```diff
@@ -50,7 +50,7 @@ func (s *autopilotMsgSuite) TestAutoPilotMessageIsPrinted(c *check.C) {
 
 	expected := "(?ms)" +
 		".*" +
-		"^The snappy autopilot is updating your system.*\n" +
+		"^snappy autoupdate is updating your system.*\n" +
```

Comment:
Do we even need a term for this?  Why isn't that saying "snappy is automatically updating your system"?